tez-mapreduce/src/test/java/org/apache/tez/mapreduce/hadoop/TestMRInputHelpers.java

```diff
@@ -56,9 +62,12 @@ public class TestMRInputHelpers {
   private static Path oldSplitsDir;
   private static Path newSplitsDir;
 
-  private static String TEST_ROOT_DIR = "target"
+  private static final String TEST_ROOT_DIR = "target"
       + Path.SEPARATOR + TestMRHelpers.class.getName() + "-tmpDir";
 
+  private static final Path LOCAL_TEST_ROOT_DIR = new Path("target"
+      + Path.SEPARATOR + TestMRHelpers.class.getName() + "-localtmpDir");
+
```

Comment:
this is creating a path & then resolving it to LocalFs, maybe just doing a ``Files.createTempDirectory`` in the test might have saved you from this multiple..

Reply:
ack, will do